utils/auth_utils.py

```diff
@@ -0,0 +1,14 @@
+from decouple import config
+from random import randint
+import requests
+
+def send_otp(contact):
+    otp = str(randint(1000, 9999))
+    url = "http://www.merasandesh.com/api/sendsms"
+    message = "Your OTP for E-Cell NIT Raipur APP is " + otp + ""
```

Comment:
It looks nice to format it with `str("Your OTP for E-Cell NIT Raipur is {}".format(otp))`

Reply:
Okay, done